cimgraph/loaders/blazegraph/query_parsers.py

```diff
@@ -49,24 +49,25 @@ def build_cim_object(conn, cim, feeder_mrid, typed_catalog:Dict, mRID_list:List[
         cls = class_name
         mRID = result['mRID']['value']
         name = result['name']['value']
-        try: 
-            class_type = eval(f"cim.{cls}")
-            #add class to typed_catalog if not already defined
-            if class_type not in typed_catalog.keys():
-                typed_catalog[class_type] = {}
+        if len(mRID) > 0:
+            try: 
+                class_type = eval(f"cim.{cls}")
```

Comment:
Consider using this paradigm instead of eval
import some_module as cim

cls = "SomeClass"
class_type = getattr(cim, cls)
instance = class_type()

Reply:
That would give a more informative error message should it fail I believe.